pkg/chains/storage/gcs/gcs.go

```diff
@@ -60,7 +62,9 @@ func NewStorageBackend(ctx context.Context, logger *zap.SugaredLogger, cfg confi
 }
 
 // StorePayload implements the storage.Backend interface.
-func (b *Backend) StorePayload(ctx context.Context, tr *v1beta1.TaskRun, rawPayload []byte, signature string, opts config.StorageOpts) error {
+func (b *Backend) StorePayload(ctx context.Context, _ versioned.Interface, obj objects.TektonObject, rawPayload []byte, signature string, opts config.StorageOpts) error {
+	// TODO: Handle unsupported type gracefully
```

Comment:
Is this a "not yet supported" unsupported, or a "won't ever support" unsupported? (if the first option, can we include a tracking issue so people can follow?)

Reply:
It's not yet supported, although there's no reason this shouldn't work. We just haven't tested this ourselves. We can add an issue to track testing the backend to make sure it works as intended. Where's the best place to create that ticket?